libraries/ui/src/Card.tsx

```diff
@@ -15,37 +21,61 @@ export const Card: React.FC<CardProps> = ({
   imageSrc,
   title,
   subtitle,
+  ctaMetadata,
   ctaUrl,
+  ctaText,
+  isEntireCardClickable = false,
   isExternalUrl = false,
   className = '',
   imageClassName = '',
 }) => {
+  const Wrapper = isEntireCardClickable && ctaUrl ? 'a' : 'div';
+
   return (
-    <div className={`card flex flex-col items-start ${className}`}>
-      <div className="card__image-container w-[323px] h-[223px] mb-3">
+    <Wrapper
+      {...(isEntireCardClickable && ctaUrl && { href: ctaUrl, ...newTabProps(isExternalUrl) })}
+      className={clsx(
+        'card flex flex-col items-start transition-transform duration-200',
+        className,
+        isEntireCardClickable && 'hover:scale-[1.01]',
+      )}
+    >
+      <div className="card__image-container max-w-[323px] max-h-[223px] w-full mb-3">
         <img
-          className={`card__image w-full h-full object-cover rounded-2xl ${imageClassName}`}
+          className={`card__image w-full max-h-full object-cover rounded-2xl ${imageClassName}`}
           src={imageSrc}
           alt={`${title}`}
         />
       </div>
 
-      <div className="card__content flex flex-col gap-8 w-full">
+      <div className="card__content flex flex-col gap-6 w-full flex-1 justify-between">
         <div className="card__text">
-          <h2 className="card__title text-2xl font-semibold text-bluedot-darker">{title}</h2>
+          <h2 className="card__title text-2xl font-semibold text-bluedot-darker mb-1">{title}</h2>
           {subtitle && (<p className="card__subtitle text-base text-black">{subtitle}</p>)}
         </div>
 
-        <CTALinkOrButton
-          url={ctaUrl}
-          variant="secondary"
-          withChevron={false}
-          isExternalUrl={isExternalUrl}
-        >
-          LinkedIn
-        </CTALinkOrButton>
+        <div className="card__cta-row flex items-center justify-between mt-auto">
+          {ctaMetadata && (
+            <p className="card__cta-metadata text-left text-xs text-black">
+              {ctaMetadata}
+            </p>
+          )}
+          {isEntireCardClickable || !ctaUrl ? (
+            <Tag className="card__cta_inert hover:bg-bluedot-lighter">{ctaText}</Tag>
+          ) : (
+            <CTALinkOrButton
+              className="card__cta"
+              url={ctaUrl}
+              variant="secondary"
+              withChevron={false}
+              isExternalUrl={isExternalUrl}
+            >
+              {ctaText}
+            </CTALinkOrButton>
```

Comment:
nice call out that we'll need to handle nested URLs! id recommend we leverage the existing handling in CTALinkOrButton to render this with or without a URL

```suggestion
            <CTALinkOrButton
              className="card__cta"
              variant="secondary"
              withChevron={false}
              url={!isEntireCardClickable && ctaUrl}
              isExternalUrl={!isEntireCardClickable && ctaUrl && isExternalUrl}
            >
              {ctaText}
            </CTALinkOrButton>
```

Reply:
The main reason I made these separate is because the team card had a different button style, but it was only in the "Desktop [Old]" designs, so I'm assuming we actually want the team card to be the same?

Anyway I've changed this to always use `<CTALinkOrButton>`, I left the styling as is for now.
![Screenshot 2025-01-30 at 11 04 17](https://github.com/user-attachments/assets/7f1cab60-dda7-4ed4-9855-e75687449855)